File: iTSoft.CRM.UI/src/app/_models/requestviewmodel.ts
import { ContactPersonMaster } from './contactPerson';
import { FollowUpDetails } from './followupdetails';
import { OrganizationMaster } from './organization';
import { RequestMaster } from './request';
import { RequestServiceDetails, RequestServiceMaster } from './requestservice';


export class RequestViewModel
{
    RequestMaster: RequestMaster;
    OrganizationMaster : OrganizationMaster;
    ContactPersonMaster : ContactPersonMaster;
    ContactPersonMasters : ContactPersonMaster[];
    RequestServiceMasters: RequestServiceMaster[];
    RequestServiceDetails: RequestServiceDetails[];
    RequestFollowup: FollowUpDetails[];

    constructor(){
        this.RequestMaster = new RequestMaster();
        this.ContactPersonMaster = new ContactPersonMaster();
        this.OrganizationMaster = new OrganizationMaster();
        this.ContactPersonMasters = [];
        this.RequestServiceMasters = [];
        this.RequestServiceDetails = [];
        this.RequestFollowup = [];

    }
}

File: iTSoft.CRM.Domain/Models/ViewModel/RequestViewModel.cs
using iTSoft.CRM.Data.Entity.Process;
using System;
using System.Collections.Generic;
using System.Text;

namespace iTSoft.CRM.Domain.Models.ViewModel
{
    public class RequestViewModel
    {
        public RequestMaster RequestMaster { get; set; }
        public List<RequestServiceMaster> RequestServiceMasters { get; set; }
        public List<RequestServiceDetails> RequestServiceDetails { get; set; }
        public List<FollowUpDetails> RequestFollowup { get; set; }
    }

}
